### PR TITLE
fix: correct typo in writing-a-vendure-plugin.md

### PR DESCRIPTION
**Pull request by seminarian**
Btw: Something else that might be helpful in this page is something about where in the file structure to put the files.

### CODE REVIEW

File: docs/content/plugins/writing-a-vendure-plugin.md [hunk 1]
```diff
@@ -141,7 +141,7 @@ export class RandomCatPlugin {}
 
 ### Step 6: Declare any providers used in the resolver
 
-In order that out resolver is able to use Nest's dependency injection to inject and instance of `CatFetcher`, we must add it to the `providers` array in our plugin:
+In order that out resolver is able to use Nest's dependency injection to inject an instance of `CatFetcher`, we must add it to the `providers` array in our plugin:
```
Comment (michaelbromley) [inline docs/content/plugins/writing-a-vendure-plugin.md]: ah there are actually 2 typos! "In order that **out** resolver" is the one I thought you were talking about. Looks like I was typing too fast on that day 😆

**Review by michaelbromley — CHANGES_REQUESTED** [review]
Spotted another typo!

File: docs/content/plugins/writing-a-vendure-plugin.md (hunk 1 above)
Comment (seminarian) [reply docs/content/plugins/writing-a-vendure-plugin.md]: Done! :)

**Comment by seminarian** [comment]
`Good suggestion! If you'd like to propose something, I'd happily accept another PR.`

Maybe later, I'm just starting learning so I'm not capable yet!

**Comment by michaelbromley** [comment]
Sure, thanks for your contribution and welcome to the Vendure community! 👍

**Pull request merged**